sql-plugin/src/main/scala/com/nvidia/spark/rapids/python/PythonConfEntries.scala

```diff
@@ -60,7 +59,9 @@ object PythonConfEntries {
   val PYTHON_POOLED_MEM = conf("spark.rapids.python.memory.gpu.pooling.enabled")
     .doc("Should RMM in Python workers act as a pooling allocator for GPU memory, or" +
       " should it just pass through to CUDA memory allocation directly. When not specified," +
-      s" It will honor the value of config '${POOLED_MEM.key}'")
+      s" It will honor the value of config '${RMM_POOL.key}', but now only 'DEFAULT' and" +
+      " 'NONE' is supported. If 'ASYNC' or 'ARENA' is specified, it will fall back to" +
```

Comment:
```suggestion
      s" It will honor the value of config '${RMM_POOL.key}', but now only 'DEFAULT' and" +
      " 'NONE' are supported. If 'ASYNC' or 'ARENA' is specified, it will fall back to" +
```